src/styles/typography.scss

```diff
@@ -87,7 +87,7 @@ $small-empty-state-height: 12px;
 }
 
 @mixin heading-padding {
-  padding: 0 4px;
+  padding: 0.5px 4px;
```

Comment:
@SergeyRoyt this looks like a "hack" what is the root issue ? maybe this changed because of Poppins ?